src/source/Ice/IceAgent.c

```diff
@@ -2327,6 +2356,7 @@ STATUS handleStunPacket(PIceAgent pIceAgent, PBYTE pBuffer, UINT32 bufferLen, PS
                       pIceCandidatePair->remote->id,
                       pIceCandidatePair->roundTripTime / HUNDREDS_OF_NANOS_IN_A_MILLISECOND);
                 pIceCandidatePair->state = ICE_CANDIDATE_PAIR_STATE_SUCCEEDED;
+                pIceAgent->rtcIceServerDiagnostics[pIceCandidatePair->local->iceServerIndex].totalRoundTripTime += pIceCandidatePair->roundTripTime;
```

Comment:
this is a duplicate with LN2329

Reply:
LN2329 is incrementing totalResponses received and this is updating totalRoundTripTime. 
I have fixed it to update both only for relay candidates